autopush/router/apns2.py

```diff
@@ -117,16 +117,14 @@ def send(self, router_token, payload, apns_id,
                 raise RouterException(
                     "APNS Transmit Error {}:{}".format(response.status,
                                                        reason),
-                    status_code=500,
+                    status_code=502,
                     response_body="APNS could not process "
-                                  "your message {}".format(reason))
+                                  "your message {}".format(reason)
+                )
         except HTTP20Error as ex:
             connection.close()
-            raise RouterException(
-                "APNS Processing error: {}".format(repr(ex)),
-                status_code=503,
-                response_body="APNS returned an error processing request",
-            )
+            self.log.error("APNS Processing error: {}".format(repr(ex)))
```

Comment:
that's fine but we could be logging the same HTTP20Error to sentry twice here, apns2's log.failure and apnsrouter's HTTP20Error clause